django_tasks/backends/database/backend.py

```diff
@@ -88,3 +91,18 @@ async def aget_result(self, result_id: str) -> TaskResult:
             return (await DBTaskResult.objects.aget(id=result_id)).task_result
         except (DBTaskResult.DoesNotExist, ValidationError) as e:
             raise ResultDoesNotExist(result_id) from e
+
+    def check(self, **kwargs: Any) -> List[CheckMessage]:
+        if "django_tasks.backends.database" not in settings.INSTALLED_APPS:
+            backend_name = self.__class__.__name__
+            package_name = sys.modules[__name__].__package__
+
+            return [
+                CheckMessage(
+                    ERROR,
+                    f"{backend_name} configured as django_tasks backend, but database app not installed",
+                    f"Insert '{package_name}' after 'django_tasks' in INSTALLED_APPS",
```

Comment:
Question: Does it _have_ to be after?

Nit: You're not currently checking that it's installed after.

Reply:
It does *not* have to come after. I removed that part of the instruction.